airbyte-webapp/src/packages/cloud/views/auth/components/Header.tsx

```diff
@@ -16,13 +16,10 @@ const Links = styled.div`
   align-items: center;
 `;
 
-const BackLink = styled.div`
+const BackLink = styled.a`
   font-style: normal;
   font-weight: bold;
-  font-size: 14px;
-  line-height: 17px;
```

Comment:
:information_source: This styles were unnecessaary, since this `div` does not contain anything besides the icon (to which those don't apply) and the `TextBlock` div below, which overwrote them. They did not affect rendering at all.